hypervisor/network/network_linux.go

```diff
@@ -790,7 +791,54 @@ func DeleteBridge(name string) error {
 	return nil
 }
 
-func AddToBridge(iface, master *net.Interface) error {
+// AddToBridge attch interface to the bridge,
+// we only support ovs bridge and linux bridge at present.
+func AddToBridge(iface, master *net.Interface, veth netlink.Link) error {
+	switch master.Name {
```

Comment:
Well, I prefer to use `ovs-vsctl port-to-br` to get the ovs bridge name in GetBridgeFromIndex. There is no need to pass veth to AddToBridge.